bin/generate_commit

```diff
@@ -135,16 +157,25 @@ function _commit_schemas() {
         --generic-schema \
         --mps-branch "$mps_branch_source" \
         --out-dir schemas
+    _create_readme "Generated Schemas" \
```

Comment:
The text here can probably be hard coded directly into the readme without need for variables.